package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "s53_memesnap",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

Comment:
It's a good practice to specify the author's name in the "author" field. This gives credit to the author and can also be useful for users who may want to reach out for questions or contributions. [medium]